lutris/services/gog.py

```diff
@@ -95,6 +95,16 @@ def __init__(self):
         super().__init__()
         self.selected_extras = None
 
+        gog_locales = {
```

Comment:
Is it possible to add more locales or are these the only ones supported by GOG extras?

Reply:
These are all locales supported by GOG.